src/main/java/frc/robot/RobotContainer.java

```diff
@@ -19,12 +20,14 @@
  * subsystems, commands, and trigger mappings) should be declared here.
  */
 public class RobotContainer {
+    public final ArmSubsystem arm = new ArmSubsystem();
```

Comment:
This doesn't need to be public
```suggestion
final ArmSubsystem arm = new ArmSubsystem();
```